### DISCUSSION TITLE
Officially supported DB versions

### DISCUSSION THREAD

**Discussion by cupcakearmy** [monicahq/monica#5368] [Q&A] [answered]
Hi there,

I wanted to ask what are the officially supported DBs and versions.
https://github.com/monicahq/monica/blob/0ae575f4a5b8571e0042ca8553e3948e23019687/docs/installation/readme.md

In the readme only mysql is specified, but in the official debian install guide maria is used. No versions are specified.
Is there an official list of supported versions?

Thanks!

**Comment by thearchivalone** [comment]
I saw that the docs say mysql/mariadb is the only one currently supported but there is an `.env` file for postgres available under `scripts/ci` so it at least tests for other dbs but your mileage may vary. If postgres does successfully work, I’ll be surprised and would definitely add Monica to my apps.

**Comment by asbiin** [comment, marked as answer]
We currently support 
- mysql > 5.7
- mysql > 8
- mariadb - but I don't know what version

**Reply by Macleykun** [reply]
May I ask why MySQL 5.6 is the preferred way for docker instead of MySQL 8>?
